lib/Federation/Proxy/TalkV1/Controller/CallController.php

```diff
@@ -30,6 +31,39 @@ public function __construct(
 	) {
 	}
 
+	/**
+	 * @see \OCA\Talk\Controller\RoomController::getPeersForCall()
+	 *
+	 * @param Room $room the federated room to get the call peers
+	 * @param Participant $participant the federated user to get the call peers
+	 * @return DataResponse<Http::STATUS_OK, TalkCallPeer[], array{}>
+	 * @throws CannotReachRemoteException
+	 *
+	 * 200: List of peers in the call returned
+	 */
+	public function getPeersForCall(Room $room, Participant $participant): DataResponse {
+		$proxy = $this->proxy->get(
+			$participant->getAttendee()->getInvitedCloudId(),
+			$participant->getAttendee()->getAccessToken(),
+			$room->getRemoteServer() . '/ocs/v2.php/apps/spreed/api/v4/call/' . $room->getRemoteToken(),
+		);
+
+		/** @var TalkCallPeer[] $data */
+		$data = $this->proxy->getOCSData($proxy);
+
+		// FIXME post-load status information of now local users
```

Comment:
You copy paste too much :P 
```suggestion
```

Reply:
Actually I keept the FIXME on purpose :shrug:  I assumed that if it had to be fixed when getting the participant list (because maybe the idea was to automatically do it when getting the OCS data, I do not know) then it would have to be fixed here as well.

Reply:
the peer list does not have the user status in non-federated conversations either, so nop.
And the normal participant list was fixed with https://github.com/nextcloud/spreed/pull/12959